app/build.gradle.kts

```diff
@@ -15,8 +15,8 @@ bugsnag {
 }
 
 android {
-    compileSdk = ProjectConfig.compileSdk
 
+    compileSdk = 34
```

Comment:
Randomly bumping the `compileSdk` of a single module in this multi module project?